sqlite/queries/select_fields_to_index.py

```diff
@@ -98,6 +98,8 @@
                         'libelle_voie',libelle_voie,
                         'libelle_voie_2',libelle_voie_2,
                         'liste_finess', liste_finess,
+                        'liste_id_bio', liste_id_bio,
+                        'statut_bio', statut_bio,
```

Comment:
Prefer structure : 
liste_id_bio at siret level. Maybe liste_bio
and move statut at siren level "VALIDE/INVALIDE"